enterprise/cmd/frontend/internal/own/resolvers/resolvers_test.go

```diff
@@ -823,3 +824,50 @@ func TestCommitOwnershipSignals(t *testing.T) {
 		},
 	})
 }
+
+func Test_SignalConfigurations(t *testing.T) {
+	// db := database.NewMockDB()
+	logger := logtest.Scoped(t)
+	db := database.NewDB(logger, dbtest.NewDB(logger, t))
+	git := fakeGitserver{}
+	own := fakeOwnService{}
+
+	ctx := context.Background()
+
+	schema, err := graphqlbackend.NewSchema(db, git, nil, graphqlbackend.OptionalResolver{OwnResolver: resolvers.NewWithService(db, git, own, logger)})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	graphqlbackend.RunTest(t, &graphqlbackend.Test{
+		Context: ctx,
+		Schema:  schema,
+		Query: `
+			query asdf {
+			  signalConfigurations {
+				name
+				description
+				isEnabled
+				excludedRepoPatterns
+			  }
+			}`,
+		ExpectedResult: `{"signalConfigurations":[{"name":"recent-contributors","description":"Indexes contributors in each file using repository history.","isEnabled":false,"excludedRepoPatterns":[]},{"name":"recent-views","description":"Indexes users that recently viewed files in Sourcegraph.","isEnabled":false,"excludedRepoPatterns":[]}]}`,
```

Comment:
Some formatting might help this read easier:

```suggestion
		ExpectedResult: `
		  {
		    "signalConfigurations":[
		      {
		        "name": "recent-contributors",
		        "description": "Indexes contributors in each file using repository history.",
		        "isEnabled": false,
		        "excludedRepoPatterns": []
		      },
		      {
		        "name": "recent-views",
		        "description": "Indexes users that recently viewed files in Sourcegraph.",
		        "isEnabled": false,
		        "excludedRepoPatterns":[]
		      }
		    ]
		  }`,
```